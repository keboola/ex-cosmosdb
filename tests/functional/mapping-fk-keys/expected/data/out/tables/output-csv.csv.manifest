{
    "columns": [
        "id",
        "level1"
    ],
    "primary_key": [],
    "incremental": false
}
