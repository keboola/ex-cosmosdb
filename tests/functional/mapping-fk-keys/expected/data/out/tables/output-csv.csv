"5716054bee6e764c94fa7aaa","4a59fa3c94f060c5060c16a37d8a08b1"
"5716054bee6e764c94fa7aab","d725966bce29c19dae4131145f26e529"
"5716054bee6e764c94fa7aac","87a69f76b73ba3008c4e800e9c66d9f1"
"5716054bee6e764c94fa7aad","a16939320e0ba853a22833ca84fca80d"
